<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ForgetPasswordWindow</class>
 <widget class="QMainWindow" name="ForgetPasswordWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>找回密码</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="labelTitle">
      <property name="text">
       <string>找回密码</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editUsername">
      <property name="placeholderText">
       <string>用户名</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editEmail">
      <property name="placeholderText">
       <string>邮箱</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="editPhone">
      <property name="placeholderText">
       <string>手机号</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttonLayout">
      <item>
       <widget class="QPushButton" name="btnReset">
        <property name="text">
         <string>找回密码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnBack">
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui> 